### PR TITLE
apache2ctl: add page

### PR DESCRIPTION
**Pull request by amitech**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/apache2ctl.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Restart the Apache daemon:
+
+`apache2ctl restart`
+
+- Check Apache configuration syntax file:
+
+`apache2ctl -t`
+
+- Print loaded Apache Modules:
```
Comment (agnivade) [inline pages/linux/apache2ctl.md]: We can make "M" to lowercase.

File: pages/linux/apache2ctl.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Stop the Apache daemon:
+
+`apache2ctl stop`
+
+- Restart the Apache daemon:
+
+`apache2ctl restart`
+
+- Check Apache configuration syntax file:
```
Comment (agnivade) [inline pages/linux/apache2ctl.md]: No need to mention the word "Apache" after 3 times. It is understood.

File: pages/linux/apache2ctl.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# apache2ctl
+
+> The CLI tool apache2ctl (alternatively apachectl)is a front end to the Apache HyperText Transfer Protocol (HTTP) server.  It is designed to help the administrator control the functioning of the Apache apache2 daemon.
```
Comment (agnivade) [inline pages/linux/apache2ctl.md]: This can be shortened. I would write it something like `CLI to administer the apache2 HTTP daemon.`

Comment (amitsharma928) [reply pages/linux/apache2ctl.md]: Thanks Agniva for suggestions. Let me resolve these and send PR again.

**Comment by amitech** [comment]
Hi @agnivade 

Please review again as my PR is still pending. Let me know if any change needed.

File: pages/linux/apache2ctl.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Stop the Apache daemon:
+
+`apache2ctl stop`
+
+- Restart the Apache daemon:
+
+`apache2ctl restart`
+
+- Check configuration syntax file:
```
Comment (agnivade) [inline pages/linux/apache2ctl.md]: "Test syntax of the configuration file" - to match with the flag `-t`. We try to include a word which starts with the flag.

File: pages/linux/apache2ctl.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`apache2ctl -t`
+
+- Print loaded modules list:
```
Comment (agnivade) [inline pages/linux/apache2ctl.md]: We can just say "List loaded modules:"

File: pages/linux/apache2ctl.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# apache2ctl
+
+> The CLI tool to administrate HTTP web server Apache.
+> This commmand comes with Debian based OS, for RHEL based OS see httpd.
+
+- Start the Apache daemon.  Give an error if it is already running:
```
Comment (agnivade) [inline pages/linux/apache2ctl.md]: s/Give/Throw. Also you have a extra leading space before Give.

File: pages/linux/apache2ctl.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+# apache2ctl
+
+> The CLI tool to administrate HTTP web server Apache.
+> This commmand comes with Debian based OS, for RHEL based OS see httpd.
```
Comment (agnivade) [inline pages/linux/apache2ctl.md]: This line can be improved slightly. @sbrl / @pxgamer - Any suggestion ?

Also, what do you mean by "RHEL based OS" ? I thought RHEL was just an OS ? Are there variants of RHEL ?

Comment (pxgamer) [reply pages/linux/apache2ctl.md]: RHEL is an OS. But it looks like there are some others derived from it's code.
https://en.m.wikipedia.org/wiki/Red_Hat_Enterprise_Linux_derivatives

As for the description, I'm not sure. :thinking:

Comment (amitech) [reply pages/linux/apache2ctl.md]: Yes @pxgamer @agnivade  there are lot of other distros which are derived from Redhat's RHEL distro like CentOS, Fedora, Scientific Linux, Amazon Linux etc. These distros use **yum** for package management. Other popular distros are based on Debian like Ubuntu, Linux Mint, Elementory OS etc which use **apt-get** for package management.

So my above written line makes sense from this point of view.

**Comment by amitech** [comment]
@pxgamer I followed 2nd point of this guildline so used prefix **sudo**. 
https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines

For CLA agreement, I am unable to get its reason, may be git global config was having some other name(on another machine) that is why some commits are showing not signed.

**Comment by pxgamer** [comment]
@amitech cheers. Obviously I need to rereed the guidelines. Looks fine to me. Lol. :+1:

**Comment by agnivade** [comment]
@amitech - This should be good to go once you take care of the CLA.

**Comment by amitech** [comment]
@agnivade - Thanks and for CLA I have followed process shared by @pxgamer and changed author on my local. But how these will be reflected on remote also. I pushed but still author is same in old commits. How can I delete old commits?

**Comment by agnivade** [comment]
force push ?

**Comment by amitech** [comment]
Thanks both @agnivade @pxgamer  . It is done :)

**Pull request merged**